### ISSUE TITLE
Add capability to start headless server

### ISSUE THREAD

**Issue by KneeAwn** [langium/langium#527]
It would be nice to be able to run generated language servers as headless servers that communicate via stdin and stdout. This would allow integration with editors like vim and emacs. The price would be sacrificing some of the benefits that a connection to vscode offers.
I believe that this would be worthwhile since Langium could be a one-stop shop for creating language servers regardless of the editors chosen by the language users.

**Comment by spoenemann** [comment]
Hi @KneeAwn! Langium itself is already independent of vscode. It's just the [Yeoman generator](https://www.npmjs.com/package/generator-langium) that produces a vscode extension, but the language server should be independent of the actual editor. All you have to do is start the generated `main` module with Node.js (`src/language-server/main.ts`, JS output in `lib/language-server/main.js`).

Could you try that and give us feedback whether you encounter any problems?

**Comment by KneeAwn** [comment]
That's perfect thank you! Is there any desire to make this a first class option via adding it to the cli automatically?

**Comment by KneeAwn** [comment]
@spoenemann I ran the command `node ./out/language/main.js --stdio` in the project and got the error that the package `vscode` could not be found. It is required in `/path/to/project/node_modules/vscode-languageclient/lib/node/main.js`. I tried installing it and still had the same error. I also noticed that when installing the `vscode` package, it says that it has been deprecated for `@types/vscode` which is already installed in the project. I'm not sure where to go from here as far as debugging.
